src/app/app-component/app.component.spec.ts

```diff
@@ -11,6 +12,7 @@ describe('AppComponent', () => {
       declarations: [
         AppComponent
       ],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA]
```

Comment:
Could you clarify please, why do you use `CUSTOM_ELEMENTS_SCHEMA`?

Reply:
Is used to tell Angular's TestBed to not throw errors on unknown elements and attributes found in the component's template.

Reply:
In this case, maybe it's better to just declare HeaderComponent in the declarations array instead of using CUSTOM_ELEMENTS_SCHEMA.

Reply:
Yes, in such cases, instead of the dependencies of the tested component, fake components and services are created.